build.sh

```diff
@@ -64,21 +65,37 @@ bootstrap_unprepare()
 
 add_update_scripts()
 {
-    local_script_dir="${CUR_DIR}/scripts"
-    target_script_dir="${ROOTFS_DIR}/sbin"
-    entrypoint="startup.sh"
-
-    if [ ! -f "${local_script_dir}/${entrypoint}" ]; then
-        echo "Missing entrypoint script '${local_script_dir}/${entrypoint}'."
-        exit 1
+    target_script_dir="${ROOTFS_DIR}${SYSTEM_UPDATE_DIR}.d"
+    if [ ! -d "${target_script_dir}" ]; then
+        mkdir -p "${target_script_dir}"
     fi
 
+    local_script_dir="${CUR_DIR}/scripts"
     for script in "${local_script_dir}"/*.sh; do
         basename="${script##*/}"
         echo "Installing ${script} on '${target_script_dir}/${basename}'."
         cp "${script}" "${target_script_dir}/${basename}"
         chmod +x "${target_script_dir}/${basename}"
     done
+
+    entrypoint_script="start_update.sh"
+    chroot "${ROOTFS_DIR}" ln -s "${SYSTEM_UPDATE_DIR}.d/${entrypoint_script}" "/sbin/${entrypoint_script}"
```

Comment:
I'm not sure if it makes sense that the start_update.sh script is located in the .d directory. As .d directories are generally for glob actions, and you don't want the entry point to be picked up by a glob rule.

Reply:
So maybe make a split of glob files and other files?

Reply:
Yes true, i will fix this in my next task if that is alright with you. In the next tast i will actually be globbing the scripts and executing them.